go/executor/cmd.go

```diff
@@ -71,8 +71,11 @@ func sqlflowCmd(cwd, driverName string) (cmd *exec.Cmd) {
 		cmd = exec.Command("docker", "run", "--rm",
 			fmt.Sprintf("-v%s:/work", cwd),
 			"-w/work", "--network=host", "-i", tfImg, "python")
+	} else if hasPython() {
```

Comment:
Maybe add comments on why we should have this branch?

Reply:
Will do it in the next PR.